src/test/java/org/mule/test/http/functional/requester/HttpRequestProxyConfigTestCase.java

```diff
@@ -101,28 +89,9 @@ public void stopMockProxy() throws Exception {
 
   @Test
   public void testProxy() throws Exception {
-    checkProxyConfig();
```

Comment:
To keep this checks that were basically making sure the config had been created with the specified values I had to expose the HttpRequestProvider and all that package. Since the SDK tests must be validating that already and I don't want to compromise the isolation testing as a whole, I decided to remove this.